stable/yourls/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
```

Comment:
@LeoColomb Should we disable this if `.Values.ingress.Certmanager=true`  ?

Reply:
Maybe? 😇 idk

https://github.com/helm/charts/blob/3b0a8f9bb14e18f25022a0a51e41fee0020c2cf4/stable/wordpress/templates/tls-secrets.yaml#L1-L18

Reply:
If cert-manager is being used, then there is no reason to provide your own certificates (its job is to automatically provision them for you)